src/main/java/reposense/parser/CsvParser.java

```diff
@@ -96,7 +96,7 @@ protected String getValueInElement(final String[] elements, int position) {
 
     /**
      * Gets the value of {@code position} in {@code elements}.
-     * Returns the value of {@code position} in an array, delimited by {@code AUTHOR_ALIAS_AND_GLOB_SEPARATOR}
+     * Returns the value of {@code position} in an array, delimited by {@code MANY_VALUES_SEPARATOR}
      * if it is in {@code element} and not empty.
      * Otherwise returns an empty array.
```

Comment:
Forgot that this returns a `List` now instead of an `Array`. do update header comments